updating/understanding-openshift-upgrade-duration.adoc

```diff
@@ -0,0 +1,91 @@
+:_content-type: ASSEMBLY
+[id="understanding-openshift-upgrade-duration"]
+= Understanding {product-title} upgrade duration
+include::_attributes/common-attributes.adoc[]
+:context: understanding-openshift-upgrade-duration
+
+toc::[]
+
+{product-title} upgrade duration varies based on the deployment topology. This section helps you understand the factors that affect upgrade duration and estimate how long the cluster upgrade duration takes in your production environment.
+
+From {product-title} 4 onward, {op-system-first} is the only supported operating system (OS) for cluster upgrades. For clusters with {op-system-base} compute nodes must be xref:../updating/updating-cluster-rhel-compute.adoc#updating-cluster-rhel-compute[updated separately].
+
+== Prerequisites
+* You are familiar with xref:../architecture/architecture.adoc#architecture[OpenShift Container Platform architecture] and xref:../updating/understanding-openshift-updates.adoc#understanding-openshift-updates[OpenShift Container Platform updates].
+
+== Factors affecting upgrade duration
+
+The following factors can affect your cluster upgrade duration:
+
+* the deployment of the target upgrade payload by the Cluster Version Operator (CVO)
+* the machine configuration applied by Machine Config Operator (MCO) to your cluster
+* the number of {op-system-base-full} worker nodes
```

Comment:
This is not correct. Because the ` maxUnavailable` and `PodDisruptionBudget` setting definitely impacts the upgrade duration.